lib/tasks/complexity.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+var complexity = require('../pipelines/complexity');
+var normalizeOptions = require('../options');
+var path = require('path');
+
+module.exports = function(options) {
+	options = normalizeOptions(options);
+	var gulp = options.gulp;
+	var taskPrefix = options.taskPrefix;
+
+	gulp.task(taskPrefix + 'complexity:report', function() {
+		return gulp.src(options.complexityGlobs)
```

Comment:
I was following the pattern set by the format and lint tasks since it falls in the same category as those.

https://github.com/metal/gulp-metal/blob/master/lib/options.js#L22